### PR TITLE
[13.0] [MIG] fieldservice_vehicle

### PR DESCRIPTION
**Pull request by brian10048**
Migration of fieldservice_vehicle to 13.0

#354

### CODE REVIEW

File: fieldservice_vehicle/readme/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+To install Field Service and have the mapping features, you need to install GeoEngine.
+
+Please refer to the installation instructions available at:
+https://github.com/OCA/geospatial/tree/12.0/base_geoengine
```
Comment (bodedra) [inline fieldservice_vehicle/readme/INSTALL.rst]:
```suggestion
https://github.com/OCA/geospatial/tree/13.0/base_geoengine
```

**Review by bodedra — APPROVED** [review]
Code review: :+1: 

Added minor comment.

File: fieldservice_vehicle/readme/INSTALL.rst (hunk 1 above)
Comment (brian10048) [reply fieldservice_vehicle/readme/INSTALL.rst]: Thank you for noticing.  Fixing it here and my other PRs where this needs updated as well

**Review by marcelsavegnago — APPROVED** [review]
Funcional Review Ok

**Pull request merged**